protos/robot/NUgus.proto

```diff
@@ -146,7 +146,7 @@ PROTO NUgus [
                 children [
                   DEF right_upper_arm NUgus_right_upper_armMesh {
                     robotTexture [
-                      %{='"./robot/NUgus_textures/NUgus_upper_arm_' .. color .. '.jpg"'}%
+                      %{='"../NUgus_textures/NUgus_upper_arm_' .. color .. '.jpg"'}%
```

Comment:
Is the `NUgus_textures` folder autogenerated? Can't seem to find it in `protos/`.

Reply:
https://github.com/NUbots/NUWebots/tree/main/protos/robot/NUgus_textures

Reply:
Ah, so it's in `protos/robot`. It's weird how webots does the relative paths. 😅